aspnetcore/fundamentals/openapi/samples/9.x/BuildTime/Skip.cs

```diff
@@ -0,0 +1,34 @@
+#if NEVER
+<!--<snippet_1>-->
+using ControllerApi.Data;
+using Microsoft.EntityFrameworkCore;
+using System.Reflection;
+
+var builder = WebApplication.CreateBuilder(args);
+
+//if (Assembly.GetEntryAssembly()?.GetName().Name != "GetDocument.Insider")
+//{
+    builder.Services.AddDbContext<ControllerApiContext>(options =>
+        options.UseSqlServer(builder.Configuration.GetConnectionString("ControllerApiContext")
+        ?? throw new InvalidOperationException("Connection string 'ControllerApiContext' not found.")));
+//}
```

Comment:
I don't see any connection string or sql info when `//if (Assembly.GetEntryAssembly()?.GetName().Name != "GetDocument.Insider")` is commented out. I don't see any difference in the generated  OpenAPI file with or without the build check.

Reply:
> I don't see any connection string or sql info when //if (Assembly.GetEntryAssembly()?.GetName().Name != "GetDocument.Insider") is commented out

I believe this is because there isn't a connection string configured in the appsettings for this project. See changes in https://github.com/dotnet/AspNetCore.Docs/commit/c86bab77b57f7071ea99e91717f6386b06ec25b0.

> I don't see any difference in the generated OpenAPI file with or without the build check.

In this case, that is the expected behavior. 👍🏽